app/views/applications/show.html.erb

```diff
@@ -27,6 +27,14 @@
   <p>No results input to search</p>
 <% end %>
 
+<% if @application.pets != [] %>
```

Comment:
good logic here, a couple ways we could do this, but I like the use of the empty array.  I also like that we found the way to make a larger text box for the adoption reason text input. Better user experience!